frontend/Dockerfile

```diff
@@ -5,10 +5,12 @@ RUN mkdir /app
 WORKDIR /app
 
 # Install app dependencies
-COPY . /app/
+COPY package.json package-lock.json ./
```

Comment:
This is typically abbreviated to package*.json

Reply:
I mean in the sense that it would make more sense at a glance, rather than having to think through a wildcard.